src/directwrite/mod.rs

```diff
@@ -238,12 +231,30 @@ impl crate::Rasterize for DirectWriteRasterizer {
     fn get_glyph(&mut self, glyph: GlyphKey) -> Result<RasterizedGlyph, Error> {
         let loaded_font = self.get_loaded_font(glyph.font_key)?;
 
-        match self.rasterize_glyph(&loaded_font.face, glyph.size, glyph.c) {
-            Err(err @ Error::MissingGlyph(_)) => {
-                let fallback_font = self.get_fallback_font(&loaded_font, glyph.c).ok_or(err)?;
-                self.rasterize_glyph(&fallback_font.create_font_face(), glyph.size, glyph.c)
-            },
-            result => result,
+        let fallback_font;
+        let (font, glyph_index) = iter::once(loaded_font)
+            .chain({
+                fallback_font =
+                    self.get_fallback_font(&loaded_font, glyph.character).map(Font::from);
+                &fallback_font
```

Comment:
At this point, why assign it in here when it's always done anyways? Might as well just declare it before the chain and just use `.chain(&fallback_font)`?

Reply:
Oh, that's not intended, I'll revert it, since we shouldn't load a font unless we have missing glyph.